examples/update-demo/3-scale.sh

```diff
@@ -21,8 +21,8 @@ set -o pipefail
 NEW_SIZE=${1:-4}
 
 export KUBE_REPO_ROOT=${KUBE_REPO_ROOT-$(dirname $0)/../..}
-export KUBECFG=${KUBECFG-$KUBE_REPO_ROOT/cluster/kubecfg.sh}
+export KUBECTL=${KUBECTL-$KUBE_REPO_ROOT/cluster/kubectl.sh}
 
 set -x
```

Comment:
An optional nit would be to be consistent about where it's set, i.e. before/after export groups. But this is getting into minutia.